dpdata/amber/sqm.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 from ..periodic_table import ELEMENTS
 
+kcal2ev = 1 / 23.06054764
```

Comment:
in `amber/md` a similar conversion constant is defined. How about write a common module for defining all conversion constants?

Reply:
Good idea! I'll develop it soon.